### ISSUE TITLE
caffe build error

### ISSUE THREAD

**Issue by sgao3** [rbgirshick/py-faster-rcnn#193]
Hello all,

I am a beginner trying to get hands-on on CNN. I to followed the readme to build the package. However, I got following error when I build caffe. Anyone knows how to solve this issue? BTW, I am working on a server, there is a caffe installed in the system directory. So this package will not refer the caffe installed in the system directory, right? 

Many thanks in advance.

Susan

caffe.cpp:(.text+0x1ab7): undefined reference to `caffe::P2PSync<float>::run(std::vector<int, std::allocator<int> > const&)'
collect2: error: ld returned 1 exit status
make: *** [.build_release/tools/caffe.bin] Error 1
make: *** Waiting for unfinished jobs....

**Comment by ericromanenghi** [comment]
May you put the content of you Makefile.config here please?

**Comment by sgao3** [comment]
Here is the Makefile.config I am using. Thank you in advance!

Susan

[make_cfg.zip](https://github.com/rbgirshick/py-faster-rcnn/files/271331/make_cfg.zip)

**Comment by ericromanenghi** [comment]
Are you using an EC2 Amazon server?

**Comment by sgao3** [comment]
No, I am using a NVIDIA devbox server.

**Comment by sbargoti** [comment]
I have come across the same error - have you been able to solve this?

**Comment by bfoust** [comment]
@ericromanenghi Yes, getting the same compilation error on an EC2 g2.2xlarge instance. (it compiles/runs fine on a local Ubuntu 14.04 installation)
